test/hotspot/jtreg/ProblemList.txt

```diff
@@ -110,6 +110,7 @@ applications/jcstress/copy.java 8229852 linux-all
 
 containers/docker/TestJcmd.java 8278102 linux-aarch64
 
+runtime/logging/BlockedLoggingTest.java 8267517 generic-all
```

Comment:
First you can't ProblemList a test using the bug id of the bug that added the test - it would need to have  a separate bug id as when that bug is fixed then you remove the entry from the problem-list.

But second you can't add a test and immediately ProblemList it. The test needs to be fixed or dropped completely. A minimal test that shows you can now use stdout/stderr with async logging, even if you can't prove the logging was actually async, would be better than nothing.